docs/news/index.qmd

```diff
@@ -2,6 +2,163 @@
 
 This page provides the release notes associated with each release of RStudio and Posit Workbench. Please contact customer support (<a href="mailto:support@posit.co">support@posit.co</a>) for questions about the described changes.
 
+## RStudio 2024.04.0
+
+**"Chocolate Cosmos"**
+
+>Date: 2024-04-29
+
+### New
+
+#### RStudio
+
+- RStudio now requires R 3.6.0 or newer. (#14210)
+- RStudio's auto-completion system now supports ggplot2 aesthetic names and data columns (#8444)
+- RStudio's auto-completion system now supports the display of the "label" attribute (#14242)
+- RStudio Desktop on Windows and Linux supports auto-hiding the menu bar (#8932)
+- RStudio Desktop on Windows and Linux supports full-screen mode via F11 (#3243)
+- RStudio Desktop now supports pasting of file paths for files copied to the clipboard (#14240)
```

Comment:
```suggestion
- RStudio Desktop on Windows and Linux supports auto-hiding the menu bar. (#8932)
- RStudio Desktop on Windows and Linux supports full-screen mode via F11. (#3243)
- RStudio Desktop now supports pasting of file paths for files copied to the clipboard. (#14240)
```

Reply:
I don't want to go through each line and add ending punctuation, so please add it throughout.